simulation/src/manager.ts

```diff
@@ -1,36 +1,50 @@
-import { Message, MessageType } from './message'
+import lodash from "lodash"
```

Comment:
You should import like this : `import deepClone from 'lodash/deepClone'. This way, you't load the whole lodash package